packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -235,6 +264,35 @@ class MT5PersonalDetailsForm extends React.Component {
                                                 name='tax_identification_number'
                                                 placeholder={localize('Tax identification number')}
                                             />
+                                            <FormSubHeader title={localize('Account opening reason')} />
+                                            <Field name='account_opening_reason'>
```

Comment:
this will remain hidden until we need to make it visible for mt5 account opening, right?

Reply:
Yes it is only visible for mt5 account opening (labuan)